webapp-django/crashstats/signature/static/signature/js/signature_tab_correlations.js

```diff
@@ -21,19 +22,72 @@ SignatureReport.CorrelationsTab = function (tabName) {
 
 SignatureReport.CorrelationsTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
 
+SignatureReport.CorrelationsTab.prototype.loadControls = function() {
+    var self = this;
+
+    var defaultProduct = $('#mainbody').data('default-product');
+    var defaultChannel = $('#mainbody').data('default-channel');
```

Comment:
And refactored with https://github.com/mozilla/socorro/pull/3531/commits/a80a3e576fef2293ae803e6f15e83b846600a73e.